apps/contracts/README.md

```diff
@@ -42,56 +32,56 @@ Manages token staking and rewards with flexible betting amounts:
-
-# Deploy staking contract
-near deploy --wasmFile target/wasm32-unknown-unknown/release/staking_contract.wasm --accountId your-staking.testnet --initFunction new --initArgs '{"reward_rate": "10", "min_stake_amount": "1000000000000000000000000", "max_stake_amount": "100000000000000000000000000"}'
+# Deploy the contract
+near deploy --wasmFile target/wasm32-unknown-unknown/release/bounty_prediction_market.wasm --accountId your-contract.testnet --initFunction new --initArgs '{"reward_rate": "10", "min_stake_amount": "1000000000000000000000000", "max_stake_amount": "100000000000000000000000000"}'
 ```
 
 ## Usage Examples
 
-### Voting Contract
+### Create a Bounty
 ```bash
-# Create a poll
-near call your-contract.testnet create_poll '{"title": "Best Feature", "description": "Vote for the best feature", "options": ["Feature A", "Feature B", "Feature C"], "duration_minutes": 1440}' --accountId your-account.testnet
-
-# Vote on a poll
-near call your-contract.testnet vote '{"poll_id": 1, "option_index": 0}' --accountId your-account.testnet
+# Create a new bounty
+near call your-contract.testnet create_bounty '{"title": "Next Big Feature", "description": "What feature should we build next?", "options": ["Feature X", "Feature Y"], "max_stake_per_user": "10000000000000000000000000", "duration_blocks": 86400}' --accountId your-account.testnet
+```
 
-# Get poll results
-near view your-contract.testnet get_poll '{"poll_id": 1}'
+### Stake on a Bounty Option
+```bash
+# Stake 5 NEAR on the first option of bounty with ID 1
+near call your-contract.testnet stake_on_option '{"bounty_id": 1, "option_index": 0}' --accountId your-account.testnet --deposit 5
 ```
 
-### Staking Contract
+### Get Bounty Information
 ```bash
-# Stake tokens (e.g., 10 NEAR)
-near call your-staking.testnet stake --deposit 10 --accountId your-account.testnet
+# Get details for a specific bounty
+near view your-contract.testnet get_bounty '{"bounty_id": 1}'
+```
 
-# Check stake info
-near view your-staking.testnet get_stake_info '{"account": "your-account.testnet"}'
+### Get User's Stake Information
+```bash
+# Get a user's stake information for a specific bounty
+near view your-contract.testnet get_participant_stake '{"account_id": "your-account.testnet", "bounty_id": 1}'
+```
 
-# Claim rewards
-near call your-staking.testnet claim_rewards --accountId your-account.testnet
+### Close a Bounty (Owner only)
+```bash
+# Close a bounty after it has expired
+near call your-contract.testnet close_bounty '{"bounty_id": 1}' --accountId your-contract.testnet
+```
 
-# Update max stake amount (owner only)
-near call your-staking.testnet update_max_stake_amount '{"new_max_amount": "200000000000000000000000000"}' --accountId your-staking.testnet
+### Get Bounty Results
+```bash
+# Get the results of a closed bounty
+near view your-contract.testnet get_bounty_results '{"bounty_id": 1}'
```

Comment:
The exact filename should be verified. It's common for Cargo to replace hyphens in the package name with underscores for the output artifact, but it's worth double-checking against the package's Cargo.toml.